.github/workflows/docs.yaml

```diff
@@ -3,15 +3,11 @@ on:
   push:
     branches:
       - "main"
-      # Uncomment here until https://github.com/envoyproxy/gateway/issues/632 has been fixed.
-      # - "release/v*"
```

Comment:
so the GHA file will live within the release branch which will be edited to run that release specific docs pipeline ?

Reply:
can infer from your below comment that this point is moot now, everything happens on `main`